listenbrainz/webserver/static/js/src/APIService.ts

```diff
@@ -83,14 +192,14 @@ export default class APIService {
     return result.user_token;
   };
 
-  /*
-    Send a POST request to the ListenBrainz server to submit a listen
-  */
   submitListens = async (
     userToken: string,
     listenType: ListenType,
     payload: Array<Listen>
   ): Promise<Response> => {
+    /*
```

Comment:
I actually moved them above the function definition consciously. It makes sense to me there because we'd want to keep comments about the actual implementation inside the function.

Reply:
Sorry for that, fixed it.